hcl2template/parser.go

```diff
@@ -374,8 +374,23 @@ func (p *Parser) parseConfig(f *hcl.File, cfg *PackerConfig) hcl.Diagnostics {
 			if moreDiags.HasErrors() {
 				continue
 			}
-			cfg.Builds = append(cfg.Builds, build)
 
+			// If we are in PAR (HCP Packer registry) mode check that only one build block has been parsed.
+			// If we've already parsed one fail because PAR does not support more than one build block.
+			// bucket is created upon the call to decodeBuildConfig.
+			if cfg.bucket != nil && len(cfg.Builds) > 0 {
+				diags = append(diags, &hcl.Diagnostic{
+					Severity: hcl.DiagError,
+					Summary:  "Multiple " + buildLabel + " blocks",
+					Detail: fmt.Sprintf("For Packer Registry enabled builds only one " + buildLabel +
```

Comment:
I don't know if I'm reading wrong but it sounds like a `,` is missing. I had a difficult time understanding it more than once. I always think the `builds` is a verb 🆘 
```suggestion
					Detail: fmt.Sprintf("For Packer Registry enabled builds, only one " + buildLabel +
```

Reply:
I agree with you, Moss ! Commas, are, great,